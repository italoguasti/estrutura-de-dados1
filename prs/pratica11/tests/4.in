100
8889158 Isabella
1933178 Pedro
3788503 Miguel
3078889 Arthur
8443444 Bernardo
8621931 Sophia
0239854 Davi
4482619 Alice
9771635 Pedro
1110771 Arthur
1207594 Pedro
9937471 Sophia
4916461 Isabella
7266805 Miguel
3423889 Helena
0238919 Alice
1451359 Bernardo
7552319 Helena
0872154 Bernardo
7988347 Gabriel
4938586 Miguel
1994413 Isabella
1937918 Miguel
8628482 Davi
8036214 Gabriel
8186809 Miguel
6872215 Pedro
3365553 Isabella
6068867 Sophia
1297605 Alice
6922186 Helena
2148350 Isabella
3327632 Arthur
8666620 Pedro
1918545 Bernardo
2814319 Davi
8933768 Gabriel
1207594 Bernardo
3858256 Gabriel
0820065 Alice
5499564 Alice
2569799 Isabella
0683343 Miguel
4037572 Bernardo
8769296 Helena
3878539 Pedro
9182478 Gabriel
4146350 Sophia
6185808 Bernardo
7418717 Pedro
0947102 Sophia
2019737 Gabriel
1254349 Miguel
3070263 Arthur
1398701 Isabella
0798764 Miguel
2336422 Gabriel
1998929 Helena
4148803 Isabella
8044194 Helena
8876203 Bernardo
3282614 Davi
7614023 Alice
8229189 Bernardo
9545506 Isabella
3453676 Bernardo
6184116 Miguel
6888044 Alice
0825657 Gabriel
3341115 Miguel
3308150 Sophia
4634933 Alice
2217187 Alice
0221004 Helena
1788575 Davi
3242812 Helena
2060437 Miguel
7111647 Helena
3634868 Pedro
7302161 Arthur
5813390 Bernardo
1373267 Gabriel
9012663 Bernardo
9630558 Miguel
9627872 Alice
2502671 Pedro
5964567 Davi
4362955 Davi
4271226 Pedro
9502479 Sophia
8850618 Sophia
5237877 Isabella
0971401 Arthur
4164782 Davi
4510194 Pedro
3644533 Helena
2936160 Arthur
4428872 Gabriel
2346104 Alice
4864618 Arthur
