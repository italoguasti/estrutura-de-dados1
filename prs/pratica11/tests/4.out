0221004 Helena
0238919 Alice
0239854 Davi
0683343 Miguel
0798764 Miguel
0820065 Alice
0825657 Gabriel
0872154 Bernardo
0947102 Sophia
0971401 Arthur
1110771 Arthur
1207594 Pedro
1207594 Bernardo
1254349 Miguel
1297605 Alice
1373267 Gabriel
1398701 Isabella
1451359 Bernardo
1788575 Davi
1918545 Bernardo
1933178 Pedro
1937918 Miguel
1994413 Isabella
1998929 Helena
2019737 Gabriel
2060437 Miguel
2148350 Isabella
2217187 Alice
2336422 Gabriel
2346104 Alice
2502671 Pedro
2569799 Isabella
2814319 Davi
2936160 Arthur
3070263 Arthur
3078889 Arthur
3242812 Helena
3282614 Davi
3308150 Sophia
3327632 Arthur
3341115 Miguel
3365553 Isabella
3423889 Helena
3453676 Bernardo
3634868 Pedro
3644533 Helena
3788503 Miguel
3858256 Gabriel
3878539 Pedro
4037572 Bernardo
4146350 Sophia
4148803 Isabella
4164782 Davi
4271226 Pedro
4362955 Davi
4428872 Gabriel
4482619 Alice
4510194 Pedro
4634933 Alice
4864618 Arthur
4916461 Isabella
4938586 Miguel
5237877 Isabella
5499564 Alice
5813390 Bernardo
5964567 Davi
6068867 Sophia
6184116 Miguel
6185808 Bernardo
6872215 Pedro
6888044 Alice
6922186 Helena
7111647 Helena
7266805 Miguel
7302161 Arthur
7418717 Pedro
7552319 Helena
7614023 Alice
7988347 Gabriel
8036214 Gabriel
8044194 Helena
8186809 Miguel
8229189 Bernardo
8443444 Bernardo
8621931 Sophia
8628482 Davi
8666620 Pedro
8769296 Helena
8850618 Sophia
8876203 Bernardo
8889158 Isabella
8933768 Gabriel
9012663 Bernardo
9182478 Gabriel
9502479 Sophia
9545506 Isabella
9627872 Alice
9630558 Miguel
9771635 Pedro
9937471 Sophia
